src/engage_client.js

```diff
@@ -22,5 +28,17 @@ const HEADERS = new Headers({
  *  example: '/tags'
  */
 export function getJSON(endpointUrl) {
-  return getResource([], HOST, endpointUrl);
+  console.log(process.env.NODE_ENV)
+  if (['production', 'devsrv'].includes(process.env.NODE_ENV) || endpointUrl !== 'agendas') {
```

Comment:
instead of creating the conditions in the generic `getJSON` method, can you create a method like `getTestJSON` and have the consumers use that instead?